go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/jmhodges/levigo
+module github.com/sattler/levigo
```

Comment:
Suggestion: undo this?

Reply:
If a maintainer is willing to merge it I will revert it. Currently the usage of my change is easier with this change